sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/ddl.scala

```diff
@@ -65,6 +65,10 @@ case class CreateTempViewUsing(
   }
 
   def run(sparkSession: SparkSession): Seq[Row] = {
+    if (provider.toLowerCase == DDLUtils.HIVE_PROVIDER) {
+      throw new AnalysisException("Currently Hive data source can not be created as a view")
```

Comment:
`Hive data source can only be used with tables, you cannot use it with CREATE TEMP VIEW USING`